node/CHANGELOG.md

```diff
@@ -197,6 +197,11 @@ All notable changes to this project will be documented in this file.  The format
 
 ## 1.4.8
 
+### Added
```

Comment:
Hmm, shouldn't this be at the top?

Reply:
Ouch, seems like automerge failed. Thankfully those are duplicated entries. Fixed